Assets/GothicVR/Scripts/Player/Menu/MenuDropdownController.cs

```diff
@@ -33,7 +33,7 @@ private void SetWorldDropdown()
             worlds.Add("Orc Temple", "orctempel.zen");
             worlds.Add("Free Mine", "freemine.zen");
 
-            worldDropdown.itemText.font = GameData.I.GothicSubtitleFont;
+            worldDropdown.itemText.font = GameData.I.EmptyFont;
```

Comment:
Let's be careful. If we set the empty font to another element as well, then we copy by reference and would change for all assignments.

Why not loading all the fonts and referencing them by name etc.?
In other words: How are Daedalus files handle fonts for menu? Do they reference them or just pick a hardcoded one.

Reply:
For now we only use one font, this remains as is, when we want to change the font we will assign another spriteAsset to the text

Reply:
Makes sense. But when you assign the empty stuff and alter it later. It is q reference and all variables will be changed. Even if they shall contain different fonts.
Maybe you can just add a few more comments to make it clear directly at the code snippets.